experimental/sdk/webpubsub-socketio-extension/src/SIO/components/web-pubsub-adapter.ts

```diff
@@ -186,15 +187,23 @@ export class WebPubSubAdapterInternal extends NativeInMemoryAdapter {
    * @param id - the socket id
    */
   public socketRooms(id: SocketId): Set<Room> | undefined {
-    throw NotSupportedError;
+    if (this.nsp.sockets.has(id)) {
```

Comment:
It seems this function is intended for local use only. So we can directly `return this.sids.get(id);` just like that in default adapter